### PR TITLE
Add search

### LINKED ISSUE #3: Search
Add an input search.

See http://api3.geo.admin.ch/examples/geoadmin_search.html for an example

Search by locations and by coordinates

### PR DESCRIPTION
**Pull request by kandre**
Fix #3

- typehead.js might be put in nodes_modules (needed for search suggestions)
- I suggest to use separate .less files (as soon as the functionality needs lots of css classes)

### CODE REVIEW

**Comment by kandre** [comment]
Search by coordinates still needed

File: index.html [hunk 1]
```diff
@@ -11,7 +11,10 @@
   <body>
     <div id="header">
       <div id="logo">Logo</div>
-      <div id="search">Search</div>
+      <div id="search-container">
```
Comment (oterral) [inline index.html]: Didn't you like the old id ? :D

File: index.html [hunk 2]
```diff
@@ -23,9 +26,11 @@
     <div id="catalog">Catalog</div>
     <script src="lib/jquery.min.js"></script>
     <script src="lib/bootstrap.min.js"></script>
+    <script src="//twitter.github.io/typeahead.js/releases/latest/typeahead.bundle.js"></script>
```
Comment (oterral) [inline index.html]: Use a version number to avoid bug on update : 

`http://twitter.github.io/typeahead.js/releases/0.10.5/typeahead.bundle.js`

Comment (loicgasser) [inline index.html]: we could have it locally...

Comment (oterral) [inline index.html]: yes maybe better

File: index.html (hunk 1 above)
Comment (kandre) [inline index.html]: You are right it is not consistent!

**Comment by kandre** [comment]
Search by coordinates will comes later in another PR

File: index.html [hunk 3]
```diff
@@ -23,9 +26,11 @@
     <div id="catalog">Catalog</div>
     <script src="lib/jquery.min.js"></script>
     <script src="lib/bootstrap.min.js"></script>
+    <script src="//twitter.github.io/typeahead.js/releases/0.10.5/typeahead.bundle.js"></script>
```
Comment (loicgasser) [inline index.html]: I think you can use `package.json` and npm for that. (same as bootstrap)
https://www.npmjs.com/package/typeahead.js

Comment (oterral) [inline index.html]: Damn we have looked for typeahead not typeahead.js . Thx you very much.

**Comment by oterral** [comment]
To do : 
- [ ]  rebase
- [ ] use npm module for typeahead
- [ ] use more less syntax for the css

**Comment by kandre** [comment]
Rebased

**Comment by kandre** [comment]
Thanks for the review

File: index.html [hunk 4]
```diff
@@ -11,28 +11,33 @@
...
+        <div id="measure">Messen</div>
       </div>
     </div>
     <div id="map"></div>
-    <div id="catalog">Catalog</div>
+    <div id="catalog">Katalog</div>
     <script src="lib/jquery.min.js"></script>
     <script src="lib/bootstrap.min.js"></script>
+    <script src="lib/typeahead.bundle.js"></script>
     <script src="http://api3.geo.admin.ch/loader.js?lang=en"></script>
```
Comment (oterral) [inline index.html]: can you put `de` here pls?

**Comment by oterral** [comment]
Almost perfect : 

    - load the loader in `de`by default
    - add `lib/typeahead*` in `.gitignore` file

**Comment by kandre** [comment]
I see, totally right !

**Comment by oterral** [comment]
and a beautiful squash in one commit for all of this :)

**Comment by kandre** [comment]
Ready for merge I think :)

**Pull request merged**